lib/include/kickcat/Mailbox.h

```diff
@@ -155,4 +161,69 @@ namespace kickcat
     };
 }
 
+namespace kickcat::mailbox::response
+{
+    class Mailbox;
```

Comment:
Why the need for the forward declaration ?

Reply:
Because AbstractMessage need it (line 171) and Mailbox needs also AbstractMessage.